alf/examples/actor_critic.py

```diff
@@ -222,12 +221,17 @@ def copy_gin_configs(root_dir, gin_files):
 
 def main(_):
     logging.set_verbosity(logging.INFO)
+
+    if FLAGS.gin_file is None:
+        gin_file = glob.glob(FLAGS.root_dir + "/*.gin")
+        assert gin_file, "No gin files are found! Please provide"
+
     if not FLAGS.play:
         copy_gin_configs(FLAGS.root_dir, FLAGS.gin_file)
-        # TODO: use the gin files under FLAGS.root_dir if FLAGS.gin_file
-        # is not specified
+        gin_file = FLAGS.gin_file
```

Comment:
if FLAGS.gin_file is None, gin_file will become None

Reply:
fixed